vendor/bat-native-ledger/src/bat/ledger/internal/endpoint/uphold/post_transaction/post_transaction.cc

```diff
@@ -38,7 +38,9 @@ std::string PostTransaction::GetUrl(const std::string& address) {
 std::string PostTransaction::GeneratePayload(
     const ::ledger::uphold::Transaction& transaction) {
   base::Value denomination(base::Value::Type::DICTIONARY);
-  denomination.SetDoubleKey("amount", transaction.amount);
+  denomination.SetStringKey(
+      "amount",
+      base::StringPrintf("%f", transaction.amount));
```

Comment:
Just checking if any old clients are sending this in the wrong format? Maybe need to make sure iOS gets this one if they recently updated to 1.15.x? cc @kylehickinson just in case.

Reply:
Snippet from the issue report:
> Important: This is regression in 1.15.x. Not reproducible on 1.14.x